src/components/FilePreview/index.ts

```diff
@@ -0,0 +1,3 @@
+export {FilePreview} from './FilePreview';
+export {getFileType} from './utils';
+export {FileType} from './types';
```

Comment:
re-export `export * from './FiltePreview'` to `components/index.ts` must be added as well